data/json/items/gun/300.json

```diff
@@ -18,5 +18,57 @@
     "barrel_length" : 2,
     "magazines": [ [ "300", [ "m2010mag" ] ] ],
     "flags": "NEVER_JAMS"
+  },
+  {
+    "id": "win70",
+    "copy-from": "rifle_manual",
+    "type": "GUN",
+    "name": "Winchester Model 70",
+    "name_plural": "Winchester Model 70",
+    "description": "The Winchester Model 70 is a bolt action sporting rifle.  It has an iconic place in American sporting culture and has been held in high regard by shooters since it was introduced in 1936.",
+    "weight": 4100,
+    "volume": 12,
+    "price": 46000,
+    "bashing": 12,
+    "to_hit": -1,
+    "material": [ "steel", "wood" ],
+    "color": "brown",
+    "skill": "rifle",
+    "ammo": "300",
+    "dispersion": 225,
+    "sight_dispersion": 60,
+    "aim_speed": 6,
```

Comment:
I just looked at the rifle_manual class in CleverRaven/master and it is 8, not 6.